.github/workflows/update-israeli-bank-scrapers.yml

```diff
@@ -0,0 +1,79 @@
+name: Update israeli-bank-scrapers
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  # Manual trigger
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: false
+        default: 'information'
+        type: choice
+        options:
+          - information
+          - debug
+          - warning
+          - critical
+      tags:
+        description: 'Purpose of Run This Workflow?'
+        required: false
+        type: string
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '16'
+
+      - name: Get old version
+        id: get_old_version
+        run: |
+          old_version=$(yarn list --pattern israeli-bank-scrapers-core --depth=0 --json --non-interactive --no-progress | jq -r '.data.trees[].name' | cut -d'@' -f2)
+          echo "old_version=${old_version}" >> $GITHUB_OUTPUT
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Upgrade israeli-bank-scrapers-core
+        run: |
+          yarn upgrade israeli-bank-scrapers-core --latest
+
+      - name: Get new version
+        id: get_new_version
+        run: |
+          new_version=$(yarn list --pattern israeli-bank-scrapers-core --depth=0 --json --non-interactive --no-progress | jq -r '.data.trees[].name' | cut -d'@' -f2)
+          echo "new_version=${new_version}" >> $GITHUB_OUTPUT
+
+      - name: Check if update is needed
+        if: ${{ steps.get_old_version.outputs.old_version == steps.get_new_version.outputs.new_version }}
+        run: |
+          echo "No update needed, stopping the workflow."
+          exit 0
+          fi
+
+      - name: Generate diff link
+        id: diff_link
+        run: |
+          echo "diff_link=https://github.com/eshaham/israeli-bank-scrapers/compare/v${{ steps.get_old_version.outputs.old_version }}...v${{ steps.get_new_version.outputs.new_version }}" >> $GITHUB_OUTPUT
+
+      - name: Create pull request
+        uses: peter-evans/create-pull-request@b1ddad2c994a25fbc81a28b3ec0e368bb2021c50
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: 'deps/israeli-bank-scrapers-core-${{ steps.get_new_version.outputs.new_version }}'
```

Comment:
OK, imagine this:

1. There is a new version for `israeli-bank-scrapers`, say `5.0.0`.
1. This pipeline create a branch `deps/israeli-bank-scrapers-core-5.0.0`
1. Create a Pull Request on this branch
1. The PR is still open
1. There is new version `5.0.1`

A new PR will be created because the new upgrade will be based on a new branch.

I think the branch must be with a constant name to ride on the same PR until it merged.